src/BlockchainWriter/Controller.test.ts

```diff
@@ -0,0 +1,30 @@
+/* tslint:disable:no-relative-imports */
+import { describe } from 'riteway'
+import { LightBlock } from '../Messaging/Messages'
```

Comment:
add linebreak between node module imports and relative imports
```suggestion

import { LightBlock } from '../Messaging/Messages'
```